opta/core/kubernetes.py

```diff
@@ -217,6 +222,82 @@ def current_image_tag(layer: "Layer",) -> Optional[str]:
     return None
 
 
+def create_namespace(layer_name: str) -> None:
```

Comment:
Should be named `create_namespace_if_not_exists`

Reply:
done